website/content/docs/runtime/interpolation.mdx

```diff
@@ -12,8 +12,24 @@ task environment variables, and certain driver fields. Runtime environment
 variables are not interpretable in constraints because they are only defined
 once the scheduler has placed them on a particular node.
 
-The syntax for interpreting variables is `${variable}`. An example and a
-comprehensive list of interpretable fields can be seen below:
+Nomad supports interpreting two classes of variables: [node attributes](#interpreted_node_vars)
```

Comment:
Duplicate sentences with the previous paragraph.

Should the previous paragraph now be deleted?

Reply:
@nh2 this PR was merged over a month ago. If you want to contribute, the best thing to do would be to open a new issue (or PR!) with any improvements.